### DISCUSSION TITLE
Reenable instant logout on the PTR.

### DISCUSSION THREAD

**Discussion by heyitsbench** [chromiecraft/chromiecraft#7060] [Suggestions]

**Comment by heyitsbench** [comment]
Also a bunch of commands should be reenabled for player use, e.g. `.appear`, `.summon`, whatever the gear repair one was, etc, etc.

**Comment by 55Honey** [comment]
Good one, cheers!
Instant logout will be back after the next restart on the PTR.
Same goes for `appear`, `summon`, various `cast` and `go creature` subcommands, as well as `unaura`.
